features/bootstrap/IsolatedProcessContext.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use Symfony\Component\Process\Process;
+
+/**
+ * Defines application features from the specific context.
+ */
+class IsolatedProcessContext implements Context, SnippetAcceptingContext
+{
+    private $lastOutput;
+
+    /**
+     * @beforeSuite
+     */
+    public static function checkDependencies()
+    {
+        chdir(sys_get_temp_dir());
+        if (!@`which expect`) {
```

Comment:
@ciaranmcnulty where is it used ? I don't see any change to the behat config in this PR to introduce the registration of this context class